src/modules/Popup/Popup.js

```diff
@@ -233,19 +240,32 @@ export default class Popup extends Component {
     return portalProps
   }
 
-  hideOnScroll = (event) => {
+  hideOnScroll = (e) => {
     this.setState({ closed: true })
     window.removeEventListener('scroll', this.hideOnScroll)
     setTimeout(() => this.setState({ closed: false }), 50)
   }
 
-  onOpen = (event) => {
-    this.coords = event.currentTarget.getBoundingClientRect()
+  handlePortalMount = (e) => {
     if (this.props.hideOnScroll) {
       window.addEventListener('scroll', this.hideOnScroll)
     }
   }
 
+  // Note: We can just pass this through but since we need a custom onOpen
+  // handler it's more clear to define the onClose handler in the same way
+  handleClose = (e) => {
+    const { onClose } = this.props
+    if (onClose) onClose(e)
+  }
+
+  handleOpen = (e) => {
```

Comment:
I know we haven't standardized it yet (#623), but it seems that should just start passing an object as the second param with at least props.  I say "at least" since sometimes it will have values from state, such as `value` or certain auto-controlled props.

Thoughts?